cEP-0024.md

```diff
@@ -0,0 +1,248 @@
+# Implement GitMate automations and plugins for coala
+
+| Metadata |                                                       |
+| -------- | ----------------------------------------------------- |
+| cEP      | 24                                                    |
+| Version  | 1.0                                                   |
+| Title    | Implement Gitmate automations and plugins for coala   |
+| Authors  | Vamshi Krishna Bommerla <mailto:bommvams72@gmail.com> |
+| Status   | Proposed                                              |
+| Type     | Process                                               |
+
+## Abstract
+
+This cEP describes the improvements to the coala workflow and the necessary
+automation in [GitMate](https://gitmate.io/) to realise it, as a part of the
+[GSoC project](https://summerofcode.withgoogle.com/projects/#5344316227059712).
+
+## Introduction
+
+[GitMate](https://gitmate.io/) is an automation tool for developers. While it
+works well, there are a number of missing features which would save time of
+coala community, for example automatically close PRs opened on master of forks.
+Many of the plugins in GitMate are general plugins, which are required by every
```

Comment:
> which are required by every organization

I'm pretty sure those plugins aren't necessarily *required* by other organizations, but rather just used a lot. It's not like the organization would cease to exist without those plugins.